### ISSUE TITLE
Usage of GraphQL fragments

### ISSUE THREAD

**Issue by VeronikaVakadinova** [manifold-systems/manifold#157]
Hello all,

Lately, I have implemented several GraphQL queries and I have the following question:

Due to the fact that three or four queries have the same output that is desired for further processing, I considered using *fragments* for cleaner code.

The fragment itself is "understood" correctly inside the queries.graphql file (meaning no syntax errors occur) but inside the Java method, the query result is no longer available.

**Should I reference somehow the fragment from inside the Java logic?**

Example: 
```
query MovieQuery($id: ID!) {
    movie(id: $id) {
        ... movieData
    }
}

fragment movieData on Movie{
     id
}
```

N.B. If I remove the fragment logic and leave the query as following, the result is retrieved fine:

```
query MovieQuery($id: ID!) {
    movie(id: $id) {
         id
    }
}
```

The Java logic for retrieving the data:

```
MovieQuery query = MovieQuery.builder(id).build();
MovieQuery.Result result = query.request(URL).post();
```

So when using fragments **result** is null, otherwise, the data is retrieved properly.

Is there something else that I should add?

Thanks in advance.

**Comment by rsmckinney** [comment]
@VeronikaVakadinova Hi.  Looks like a regression in the GraphQL manifold -- the fragment is missing from the request, it should be sent alongside the query.  Fix coming soon.  Thanks for reporting this!

**Comment by VeronikaVakadinova** [comment]
@rsmckinney Thank you for the quick response.

**Comment by rsmckinney** [comment]
Fixed in 2020.1.2 via https://github.com/manifold-systems/manifold/commit/896f5b3aecb4a40109dcb4f769076ec64993f650.

>Note, a new version of the manifold IntelliJ plugin is awaiting JetBrains approval, usually within 48 hours of submission.  Please wait to use Manifold ver 2020.1.2 with the pending plugin release.
